src/external/dde-dock-plugins/disk-mount/diskmountplugin.cpp

```diff
@@ -30,6 +30,10 @@ DiskMountPlugin::DiskMountPlugin(QObject *parent)
       tipsLabel(new TipsWidget),
       diskPluginItem(new DiskPluginItem)
 {
+    QTranslator *translator = new QTranslator(this);
+    translator->load(QString("/usr/share/dde-file-manager/translations/dde-file-manager_%1.qm").arg(QLocale::system().name()));
```

Comment:
disk-mount used by dde-dock and It is inappropriate to directly use dde-file-manager  translation file here .

Reply:
this case has been solved by the latest internal submission, so this case can be closed now, and thank you for your pr and suggestions.

Reply:
ok, I have close the pr!